package.json

```diff
@@ -7,33 +7,42 @@
   "repository": "alex-e-leon/johnny-dependency",
   "author": "alex leon <alex@leon.st>",
   "license": "MIT",
+  "files": [
+    "lib"
+  ],
   "scripts": {
     "build": "babel src/ -d lib/ && chmod 755 lib/cli/index.js",
-    "start": "yarn run build && ./lib/cli/index.js",
-    "prepublish": "yarn run build",
-    "lint": "./node_modules/.bin/eslint src/",
-    "test": "yarn run flow && yarn run lint"
+    "start": "yarn build && ./lib/cli/index.js",
+    "prepare": "yarn build",
+    "lint": "eslint src test",
+    "test": "flow && yarn lint && nyc ava",
+    "test:watch": "ava --watch",
+    "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov"
```

Comment:
Is there really a point to including coverage if we're just doing snapshot testing? 😛

Reply:
Snapshot testing doesn't imply 100% coverage? It's just an easy way of writing assertions

Reply:
True